adafruit_thermal_printer/thermal_printer_2168.py

```diff
@@ -68,7 +73,7 @@ def __init__(
             auto_warm_up=auto_warm_up,
         )
 
-    def warm_up(self, heat_time=120):
+    def warm_up(self, heat_time: float = 120) -> None:
```

Comment:
It looks like this is supposed to mimic the method of the same name in the base class, albeit with a fake `heat_time` argument.  Let's typing that as `int` to remain consistent though.